proposed/dotnet-bootstrapper-cli-experience.md

```diff
@@ -0,0 +1,256 @@
+# DNVM / Dotnet Bootstrapper End-to-End CLI Experience
+
+Initial Context:
+
+The officially supported `DNVM` and `DNX` project was replaced by the `.NET CLI` as `dotnet` in `2017`.
+
+As of 2025, `DNVM` is an unofficial product, but internally developed and leveraged.
+
+We are proposing an official `DNVM` product (name subject to change).
+This is an offering similar to `nvm` or `rustup` but for `.NET`.
+There have already been many internal documents/discussions. This is a 'strawman proposal' for us to dig at further. For all mentions of the term `dnvm`, this is subject to change to an applicable product name.
+
+The `dnvm` product will be AOT bundled alongside the apphost to enable shipment via an external standalone executable, that is also callable from within the .NET SDK. The implementation details of this will be settled and discussed within another document, but it's necessary to remark on this here to explain the usage of 'dotnet' as a root command.
```

Comment:
> The `dnvm` product will be AOT bundled alongside the apphost to enable shipment via an external standalone executable

For `dnx` [it appears the plans](https://github.com/dotnet/designs/blob/main/accepted/2025/direct-tool-execution.md#dnx-summary) are to ship a script that then invokes dotnet CLI commands. Would it be a good idea to implement `dnvm` similarly?